### ISSUE TITLE
Tags page not working properly

### ISSUE THREAD

**Issue by AgtLucas** [andersnoren/beaumont#1]
Hi, @andersnoren!

First of all, amazing work you did with these WP themes, really, THANK YOU!

So, I was testing Beaumont, and I noticed that when you click on a tag, let's say "books", the tags page will show the list of all the posts, including the posts that aren't using the tag "books".

I'm not sure if it's a theme issue or what, but I tested using Björk and it's working fine.

Unfortunately I don't have a demo to show the issue yet.

Thank you! :)

**Comment by andersnoren** [comment]
Hi @AgtLucas,

Glad you like them! I can view the tag archive pages without any issues, and it's only showing the posts tagged with that tag for me. Are you running the latest version of WordPress, and have you made any changes to the Templates in the Site Editor? If so, it's possible that the default query has been overwritten.

— Anders

**Comment by AgtLucas** [comment]
@andersnoren 

Yep, I was running the latest version of WordPress (I was using the .com one, not the .org). 

I think I've made some changes to the theme, but not for that template itself... there the only thing I changed was the amount of posts to be showed.

But anyways, I decided to go with Björk, it's also amazing: https://lucas.art/ 😄 

Thank you!
